index.php

```diff
@@ -0,0 +1,138 @@
+<?php declare(strict_types=1);
+
+require __DIR__ . '/vendor/autoload.php';
+require __DIR__ . '/config/db.global.php';
+
+use Aws\TranscribeService\TranscribeServiceClient;
+use Doctrine\DBAL\Connection;
+
+Dotenv\Dotenv::create(__DIR__)->load();
+
+return function ($event) use ($conn) {
+
+    // Create Amazon Transcribe Client
+    $awsTranscribeClient = new TranscribeServiceClient([
+        'region' => $_ENV['AWS_REGION'],
+        'version' => $_ENV['AWS_VERSION']
+    ]);
+
+    // Retrieve the transcription job
+    $transcriptionJob = $awsTranscribeClient->getTranscriptionJob([
+        'TranscriptionJobName' => $event['detail']['TranscriptionJobName']
+    ]);
+
+    // parse the job to get the File Uri
+    $transcriptionRawResult = $transcriptionJob->toArray();
+
+    // get the result file from S3 URL
+    $resultFile = file_get_contents($transcriptionRawResult['TranscriptionJob']['Transcript']['TranscriptFileUri']
```

Comment:
What happens if there is no file? Perhaps its a good place to add some error handling?

Reply:
Added. Thanks for seeing that.